packages/libs/error-stack/build.rs

```diff
@@ -18,9 +18,15 @@ fn main() {
         println!("cargo:rustc-cfg=rust_1_65");
     }
 
+    #[cfg(feature = "futures")]
+    println!(
+        "cargo:warning=The `futures` feature for `error-stack` is deprecated as it's not required \
+         anymore"
+    );
+
```

Comment:
Maybe we should mention a version where we're going to remove `hooks` and `futures`?

Reply:
I added this in 0a8b49546279404d8af553fd7671bbef06f0236c